subworkflows/local/bam_variant_calling_tumor_only_all/main.nf

```diff
@@ -115,7 +115,7 @@ workflow BAM_VARIANT_CALLING_TUMOR_ONLY_ALL {
             cram.map{ meta, cram, crai ->
                 joint_mutect2 ?
                 [ meta + [ id:meta.patient ] - meta.subMap('sample', 'status', 'num_intervals', 'data_type', 'patient') , cram, crai ] :
-                [ meta - meta.subMap('sample', 'status', 'num_intervals', 'data_type'), cram, crai ]
+                [ meta + [ id:meta.sample ]- meta.subMap('sample', 'status', 'num_intervals', 'data_type'), cram, crai ]
```

Comment:
```suggestion
                [ meta + [ id:meta.sample ] - meta.subMap('sample', 'status', 'num_intervals', 'data_type'), cram, crai ]
```

Slightly confused why num_intervals a) is in there in hthe first place b) why it is removed. Don't you need it later to do proper merging across intervals or are you adding it back in then?